
package it.eng.sil.myauthservice.rest.client.sms.soap;

import javax.xml.ws.WebFault;


/**
 * This class was generated by Apache CXF 3.1.7
 * 2020-07-09T15:03:29.768+02:00
 * Generated source version: 3.1.7
 */

@WebFault(name = "InvioSMSFault", targetNamespace = "http://www.infotn.it/SMS-EAI/InvioSMS_message")
public class SendSMSFaultMessage extends Exception {
    
    private it.eng.sil.myauthservice.rest.client.sms.soap.InvioSMSFaultType invioSMSFault;

    public SendSMSFaultMessage() {
        super();
    }
    
    public SendSMSFaultMessage(String message) {
        super(message);
    }
    
    public SendSMSFaultMessage(String message, Throwable cause) {
        super(message, cause);
    }

    public SendSMSFaultMessage(String message, it.eng.sil.myauthservice.rest.client.sms.soap.InvioSMSFaultType invioSMSFault) {
        super(message);
        this.invioSMSFault = invioSMSFault;
    }

    public SendSMSFaultMessage(String message, it.eng.sil.myauthservice.rest.client.sms.soap.InvioSMSFaultType invioSMSFault, Throwable cause) {
        super(message, cause);
        this.invioSMSFault = invioSMSFault;
    }

    public it.eng.sil.myauthservice.rest.client.sms.soap.InvioSMSFaultType getFaultInfo() {
        return this.invioSMSFault;
    }
}
